packages/credential-status/src/credential-status.ts

```diff
@@ -0,0 +1,22 @@
+import { IAgentContext, IAgentPlugin, IPluginMethodMap } from '@veramo/core';
+import { Status, StatusMethod } from 'credential-status';
+import { ICheckCredentialStatusArgs } from './types';
+
+export class CredentialStatusPlugin implements IAgentPlugin {
+  private readonly status: Status;
+  readonly methods: IPluginMethodMap;
```

Comment:
I think this should be ICheckCredentialStatus

```suggestion
  readonly methods: ICheckCredentialStatus;
```

Reply:
Makes sense. Fixed.